packages/flutter_tools/lib/src/base/user_messages.dart

```diff
@@ -74,8 +74,7 @@ class UserMessages {
       'Install Android Studio from: https://developer.android.com/studio/index.html\n'
       'On first launch it will assist you in installing the Android SDK components.\n'
       '(or visit ${_androidSdkInstallUrl(platform)} for detailed instructions).\n'
-      'If the Android SDK has been installed to a custom location, set $envKey to that location.\n'
-      'You may also want to add it to your PATH environment variable.\n';
+      'If the Android SDK has been installed to a custom location, please use `flutter config --android-studio-dir` or `flutter config --android-sdk` to set $envKey to that location.\n';
```

Comment:
minor formatting: we generally try to keep our line widths to 80 characters. See examples above (the lines without the \n at the end) for breaking up a string into concatenated multi-line string

Reply:
Understood.